### PR TITLE
[RU] Translate supervisor additions

### PR DESCRIPTION
**Pull request by brain-geek**
Original change - #716 .
Translation checklist - #756 .

### CODE REVIEW

File: ru/lessons/advanced/otp-supervisors.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ children = [
 
 + `:simple_one_for_one` - Наилучшая для динамически прикрепляемых процессов стратегия. Спецификация супервизора обязана содержать описание только одного дочернего процесса, но он может быть запущен несколько раз. Предполагается, что эта стратегия используется, когда нужно динамически запускать и останавливать дочерние процессы.
 
+### Подход к перезапуску
```
Comment (nscyclone) [inline ru/lessons/advanced/otp-supervisors.md]: Probably `Опции перезапуска` or `Варианты перезапуска` would fit better in this place. What do you think, @brain-geek?

File: ru/lessons/advanced/otp-supervisors.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ children = [
...
+
+Есть несколько вариантов реализации перезапуска после падения дочернего процесса:
+
++ `:permanent` - Дочерний процесс всегда перезапускается.
+
++ `:temporary` - Дочерний процесс никогда не перезапускается.
+
++ `:transient` - Дочерний процесс перезапускается только в случае аномального завершения.
+
+Это опциональная опция, значение по умолчанию - `:permanent`.
```
Comment (nscyclone) [inline ru/lessons/advanced/otp-supervisors.md]: I'd replace `опциональная` with `необязательная`. Also `&mdash;` should be used instead of `-`.

**Comment by brain-geek** [comment]
Thanks for the feedback, @nscyclone !

I've applied some changes to address those.

**Comment by nscyclone** [comment]
Thanks, @brain-geek!

**Pull request merged**